python/src/StationaryFunctionalCovarianceModel_doc.i.in

```diff
@@ -0,0 +1,49 @@
+%feature("docstring") OT::StationaryFunctionalCovarianceModel
+"Functional covariance function.
+
+Allows to define an 1-d stationary covariance model from a custom correlation function.
+
+Parameters
+----------
+scale : sequence of floats
+    Scale coefficient :math:`\vect{\theta}\in \Rset^n`.
+    The size of :math:`\vect{\theta}` is the input dimension.
+    Must be of size equal to 1.
```

Comment:
```suggestion
```
Isn't it the output that must have dimension 1? Why should the size of theta be restricted to 1?